examples/config/broker/runtime.properties

```diff
@@ -15,8 +15,8 @@
 # limitations under the License.
 #
 
-druid.host=localhost
-druid.port=8080
+#druid.host=localhost
```

Comment:
Can we specify that in the documentation somehow? either as a comment immediately above these lines, or in the standard doc on running/configuring examples?